devlib/target.py

```diff
@@ -1035,6 +1035,18 @@ def dump_logcat(self, filepath, filter=None, append=False, timeout=30):  # pylin
     def clear_logcat(self):
         adb_command(self.adb_name, 'logcat -c', timeout=30)
 
+    def adb_reboot_bootloader(self):
+        adb_command(self.adb_name, 'reboot-bootloader', timeout=30)
```

Comment:
Actually, now that I think about it, exposing it may be the right thing to do. My thinking was -- given that this is bootloader, rather than a full system boot, it's hard to imagine any system where booting would take longer. However, this time needs to account for the system shutdown before the reboot as well. While still unlikely to be over 30 seconds, it's less unreasonable to expect that there is a platform out there where that might be the case. So yes, adding it as a parameter with a default makes sense.